core/controllers/topic_editor.py

```diff
@@ -163,14 +162,17 @@ def get(self, topic_id):
             raise self.PageNotFoundException(
                 Exception('The topic with the given id doesn\'t exist.'))
 
-        skill_ids = topic.get_all_skill_ids()
-
         skill_id_to_description_dict = (
-            skill_services.get_skill_descriptions_by_ids(topic_id, skill_ids))
+            topic_fetchers.get_descriptions_of_linked_skills(topic_id))
+
+        skill_id_to_rubrics_dict = (
+            topic_fetchers.get_rubrics_of_linked_skills(topic_id)
+        )
 
         self.values.update({
             'topic_dict': topic.to_dict(),
-            'skill_id_to_description_dict': skill_id_to_description_dict
+            'skill_id_to_description_dict': skill_id_to_description_dict,
```

Comment:
Can we merge this into one dictionary (skill_id to <description, rubrics>)? Unless that complicates things in the frontend?

Reply:
In the frontend, the description and rubrics need to be present as 2 separate dicts keyed by skill ID. So, combining them here, will lead to splitting them up at the frontend anyway. So,  I don't think this is necessary, what do you think?

Reply:
Got it that makes sense!